lib/rock_rms/resources/recurring_donation.rb

```diff
@@ -3,12 +3,25 @@ class Client
     module RecurringDonation
       def list_recurring_donations(options = {})
         res = get(recurring_donation_path, options)
-        RockRMS::RecurringDonation.format(res)
+        RockRMS::Responses::RecurringDonation.format(res)
       end
 
       def find_recurring_donation(id)
         res = get(recurring_donation_path(id))
-        RockRMS::RecurringDonation.format(res)
+        RockRMS::Responses::RecurringDonation.format(res)
+      end
+
+      def update_recurring_donation(
```

Comment:
adding a spec for this?

Reply:
Aight specs added for `#update_recurring_donation`